packages/frontend-2/components/form/TextInput.vue

```diff
@@ -26,8 +25,8 @@
         :type="type"
         :name="name"
         :class="[
-          'block w-full shadow-sm rounded-md focus:outline-none sm:text-sm text-black',
-          'disabled:cursor-not-allowed  disabled:bg-background-2 disabled:text-foreground-3',
+          'block h-12 w-full rounded-xl focus:outline-none sm:text-sm bg-background-base text-foreground transition-all',
+          'disabled:cursor-not-allowed disabled:bg-neutral-50 disabled:dark:bg-neutral-800 disabled:text-neutral-100 disabled:dark:text-neutral-400',
```

Comment:
I see "neutral" used in many places across multiple components and with more than 3 different weights, feels like these should be semantic and the set of variations should be reduced